src/stories/30_Visual_Effects/Ken_Burns.html

```diff
@@ -0,0 +1,143 @@
+</head>
+<body>
+  <amp-story standalone>
+    <!-- ## Ken Burns Effect-->
+    <!--The general idea is to nest elements to combine the animations: the container element will animate the zoom, and the image inside the container will be animated with a panning.
+* Start by creating a div which will act as the image container. Set its style class to `position:absolute`. This will remove the padding so that the image takes up the whole screen.
+* Set its animation to be `animate-in="zoom-in"` or `animate-in="zoom-out"` and its duration with `"animate-in-duration=30s"`.
+  
+
+<div class="ampstart-card info">
+**Tip:** In this demo we use 30 seconds but your mileage may vary. Try experimenting with different values to see what works best for your desired results.
+</div>
+
+* Inside this container, add an `<amp-img>` element with the `src` attribute pointing at your image, describe the size of the image with the `width` and `height` attributes of the `<amp-img>` element. Set `layout="fixed"` so that the image outsizes the screen real estate (We want to use a large image that can be panned over. We recommend 1600 x 1200 px or larger).     
+* Animate the `<amg-img>` element using the `animate-in="pan-right"`. (You can also use `pan-left`, `pan-up`, and `pan-down`. [See amp-story animations list.](/stories/features/animations/#zoom-in) 
+
+  
+<div class="ampstart-card info">
+**Tip:** It's important that the **image** sets the **panning animation**, whereas the image **container** sets the **zoom animation**. Otherwise the dimensions of the image will change, causing undesired results in the animation.
+</div>
+    -->
+    <amp-story-page id="ken-burns-effect1">
+      <amp-story-grid-layer template="vertical">
+        <div animate-in="zoom-in" animate-in-duration="30s" class="img-container">
+            <amp-img id="ken-burns-img1"
+              src="https://picsum.photos/1600/1200?image=1077"
+              animate-in="pan-left"
+              animate-in-duration="30s"
+              layout="fixed"
+              width="1600"
+              height="1200">
+            </amp-img>
+        </div>
+      </amp-story-grid-layer>
+    </amp-story-page>
+
+    <!-- ## Using Multiple Images -->
+    <!-- 
+      You can also achieve the effect with multiple images!
```

Comment:
I had to see the demo to understand what was going on. Maybe change the intro to: 

"You can even implement a slideshow transitioning between multiple pages using this effect."